ipaserver/install/cainstance.py

```diff
@@ -2208,6 +2209,44 @@ def import_ra_key(custodia):
     CAInstance.configure_agent_renewal()
 
 
+def has_ipa_ca_san(cert):
```

Comment:
Bit of a nit, but "has_x" sounds like it would return `bool`.  If you're going to raise on error, maybe "check_x" or "ensure_x" would be better?

Reply:
Good point, switched to check_